src/windowUtils.ts

```diff
@@ -1,5 +1,19 @@
-import { window, Terminal } from 'vscode';
+import { window, Terminal, TerminalOptions } from 'vscode';
+import * as path from 'path';
+import { Platform } from './platform';
+
+const toolsLocation = path.resolve(Platform.getUserHomePath(), '.vs-openshift');
  
-export function createTerminal(name: string, cwd: string, env: NodeJS.ProcessEnv = process.env ): Terminal {
-    return window.createTerminal(name, process.platform === 'win32'? undefined : '/bin/bash');
+export function createTerminal(name: string, cwd: string, env: NodeJS.ProcessEnv = process.env): Terminal {
+    let finalEnv: NodeJS.ProcessEnv = {};
+    Object.assign(finalEnv, env);
+    finalEnv.PATH = `${toolsLocation}:${env.PATH}`;
```

Comment:
It won't work on windows. `PATH` should be `Path` at least in `finalEnv`

Reply:
Right, also forgot the delimiter is different.